0-bootstrap/modules/jenkins/variables.tf

```diff
@@ -111,8 +137,32 @@ variable "sa_enable_impersonation" {
   default     = false
 }
 
+variable "storage_bucket_prefix" {
+  description = "Name prefix to use for storage buckets."
+  type        = string
+  default     = "bkt"
+}
+
+variable "storage_bucket_labels" {
+  description = "Labels to apply to the storage bucket."
+  type        = map(string)
+  default     = {}
+}
+
 variable "folder_id" {
   description = "The ID of a folder to host this project"
   type        = string
   default     = ""
 }
+
+variable "terraform_version" {
```

Comment:
will these be used by the startup scripts for the agent?

Reply:
Yes. I added a template to use this now. Thanks